tests/src/test/scala/it/unibo/scafi/test/functional/stdlib/TestTimeUtils.scala

```diff
@@ -0,0 +1,157 @@
+package it.unibo.scafi.test.functional.stdlib
+
+import it.unibo.scafi.test.FunctionalTestIncarnation._
+import it.unibo.scafi.test.functional.ScafiAssertions.assertNetworkValues
+import it.unibo.scafi.test.functional.ScafiTestUtils
+import it.unibo.utils.StatisticsUtils.stdDev
+import org.scalatest._
+
+import scala.concurrent.duration._
+import scala.math.Numeric.BigDecimalAsIfIntegral.mkOrderingOps
+
+/*
+Still to test:
+- cyclicFunction
+- cyclicFunctionWithDecay
+ */
+class TestTimeUtils extends FlatSpec{
+  import ScafiTestUtils._
+
+  val Time_Utils = new ItWord
+
+  private[this] trait SimulationContextFixture {
+    val net: Network with SimulatorOps = manhattanNet(side = 3, southEastDetached = true)
+  }
+
+  private[this] trait TestProgram extends AggregateProgram with StandardSensors with BuildingBlocks
+
+  def unitaryDecay: Int => Int = _ - 1
+  def halving: Int => Int = _ / 2
+
+  Time_Utils should "support timerLocalTime" in new SimulationContextFixture {
+    val testProgram: TestProgram = new TestProgram {
+      override def main(): Any = timerLocalTime(1 second)
+    }
+
+    exec(testProgram, ntimes = someRounds)(net)
+    net.valueMap[Long]().forall(e => e._2 > 0)
```

Comment:
**Warning!!!** This is **not** an assertion! 
(same also applies to various occurrences of `valueMap`)